docs/users/cli-usage.md

```diff
@@ -42,7 +42,7 @@ Quickstart instructions regarding database imports can be found under [Database
 
 ### Drupal 6/7 Quickstart
 
-Beginning to use ddev with a Drupal 6 or 7 project is as simple as cloning the project's repository and checking out its directory.
+Beginning to use ddev with a Drupal 6 or 7 project is as simple as cloning **your** project's repository and checking out its directory.
```

Comment:
I think the use of "your" instead of "the" is appropriate. I think your *emphasis* with double-asterisks, while explaining the reason for the PR, isn't as useful to the general user. So let's just go with "your" in all these cases instead of `**your**`` - I assume you were thinking there was ambiguity and thought people might have checked out the ddev or TYPO3 repo?

Reply:
the problem was that **the** suggested that there is an dedicated ddev repository which shoul be used. **your** indicates clearly, that i have to use my own project repository